Engine/EditorUI/Panel/PanelPopups.cpp

```diff
@@ -36,6 +36,7 @@ void PanelPopups::RenderAssetsLoadingPopup()
 	{
 		ImGui::Text("Loading Assets");
 		float progress = (float)App->resources->thread_comunication.loaded_items / App->resources->thread_comunication.total_items;
+		//assert(progress <= 1);
```

Comment:
Why is this code comented?

Reply:
This is failing because number of filesis computed incorrectly, need to be fixed